src/features.ts

```diff
@@ -0,0 +1,27 @@
+// Bridges
+export const nativeBridgeEnabled = true;
+export const layerZeroBridgeEnabled = true;
+export const layerSwapBridgeEnabled = true;
+export const celerBridgeEnabled = true;
+export const omniBridgeEnabled = true;
+export const stargateBridgeEnabled = true;
```

Comment:
Yes, variables can be private since they are used only by `isBridgeEnabled` function
Fixed in https://github.com/AstarNetwork/astar-apps/pull/1362/commits/060d2813b258aae31a1cd81b69382775dd4b6b7f